Mesh_3/include/CGAL/Mesh_domain_with_polyline_features_3.h

```diff
@@ -604,19 +604,19 @@ class Mesh_domain_with_polyline_features_3
   template <typename OutputIterator>
   OutputIterator get_curve_segments(OutputIterator out) const;
 
-  /// Return the length of the arc of curve, on the curve with index
+  /// Returns the length of the arc of curve, on the curve with index
```

Comment:
I have changed the wording to
> Returns the length of the curve segment, on the curve with index \c  curve_index, from \c p to \c q

in https://github.com/CGAL/cgal/pull/2315/commits/ce0c1bd0f11daedd3e076e379a221f3223d7e06d.